who-metrics-ui/src/components/RepositoriesTable.tsx

```diff
@@ -31,18 +31,19 @@ import {
   useRef,
   useState
 } from 'react';
+
+import { RepositoryResult } from '../../../ts-backend/src';
```

Comment:
Replaced `type Repo = (typeof repos)[0];` with the backend type since we define this already. This is more likely to be correct since `(typeof repos)[0]` may be missing certain properties.

Reply:
Good question - I _think_ it should be okay since the whole project gets cloned before we run the next build. Good point though that this could cause maintenance issues at some point. It might make sense to extract this to a shared location or something to avoid reaching up and down...